src/action.ts

```diff
@@ -1,9 +1,11 @@
 import * as core from '@actions/core'
 import {check_directory, check_urls} from './check'
+import {set_up_configuration} from './config'
 
 export async function main(): Promise<void> {
     try {
         const repositories: string = core.getInput('repositories')
+        await set_up_configuration()
```

Comment:
Once you have configuration related variables, you need to use them as an argument in set_up_configuration
like
```typescript
await set_up_configuration(curations, rules,classifications)
```